platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -70,6 +72,13 @@
  */
 public final class MerkleDb {
 
+    /**
+     * Since {@code com.swirlds.platform.Browser} populates settings, and it is loaded before any
+     * application classes that might instantiate a data source, the {@link ConfigurationHolder}
+     * holder will have been configured by the time this static initializer runs.
+     */
+    private static final MerkleDbConfig config = ConfigurationHolder.getConfigData(MerkleDbConfig.class);
```

Comment:
`config` is not used in this class.

Reply:
It's was used in `getCompactionThreadNumber()` but I removed it

Reply:
Yes. I removed it.